src/common/selftest/selftest_MINI.cpp

```diff
@@ -235,6 +250,21 @@ void CSelftest::phaseFinish() {
     disable_all_steppers();
 }
 
+bool CSelftest::phaseWait() {
+    static uint32_t tick = 0;
+    if (tick == 0) {
+        tick = m_Time;
+        return true;
+    } else if ((m_Time - tick) < 2000)
+        return true;
+    tick = 0;
+    if (m_pFSM) {
```

Comment:
this if is not really necessary, because `delete nullptr` does nothing per definition (I know it is hard to believe ;) )

Reply:
i know this, but sometimes automaticaly type "if !=nullptr". Fixed.